RadixWallet/Features/AddLedgerFactorSource/AddLedgerFactorSource+View.swift

```diff
@@ -110,39 +110,6 @@ extension View {
 	}
 }
 
-// MARK: - LinkConnectorView
-@MainActor
-struct LinkConnectorView: SwiftUI.View {
```

Comment:
unused view?

Reply:
Yes, we use an alert for this, but I made a note of the code being removed since I think we might want to go back to a screen.